listplayer.go

```diff
@@ -126,8 +133,8 @@ func (lp *ListPlayer) TogglePause() error {
 	return getError()
 }
 
-// MediaState returns the state of the current media.
-func (lp *ListPlayer) MediaState() (MediaState, error) {
+// GetMediaState returns the state of the current media.
+func (lp *ListPlayer) GetMediaState() (MediaState, error) {
```

Comment:
I noticed this method `MediaState` returned a Type `MediaState`, so I changed the method name